proc/variables.go

```diff
@@ -442,6 +447,13 @@ func (scope *EvalScope) packageVarAddr(name string) (*Variable, error) {
 	return nil, fmt.Errorf("could not find symbol value for %s", name)
 }
 
+func (v *Variable) toFieldNilChecked(field *dwarf.StructField, name string) (*Variable, error) {
+	if v.Addr == 0 {
```

Comment:
Is there any reason not to simply check this by default and fold this code into `toField` itself?